cmd/gnodev/test.go

```diff
@@ -298,17 +292,8 @@ func gnoTestPkg(
 			if err != nil {
 				errs = multierr.Append(errs, err)
 				io.ErrPrintfln("--- FAIL: %s (%s)", testName, dstr)
-				if verbose {
-					stdouterr, err := closer()
-					if err != nil {
-						panic(err)
-					}
```

Comment:
The panic here is now printed from above in a the general case.
I don't think we need this code anymore.